cmd/aro/update_ocp_versions.go

```diff
@@ -33,57 +38,111 @@ func getInstallerImageDigests(envKey string) (map[string]string, error) {
 	if !env.IsLocalDevelopmentMode() {
 		jsonData, err = base64.StdEncoding.DecodeString(string(jsonData))
 		if err != nil {
-			return nil, fmt.Errorf("%s: Failed to decode base64: %v", envKey, err)
+			return fmt.Errorf("%s: Failed to decode base64: %v", envKey, err)
 		}
 	}
 
-	if err = json.Unmarshal(jsonData, &installerImageDigests); err != nil {
-		return nil, fmt.Errorf("%s: Failed to parse JSON: %v", envKey, err)
+	if err = json.Unmarshal(jsonData, envData); err != nil {
+		return fmt.Errorf("%s: Failed to parse JSON: %v", envKey, err)
 	}
 
-	return installerImageDigests, nil
+	return nil
 }
 
-func getLatestOCPVersions(ctx context.Context, log *logrus.Entry) ([]api.OpenShiftVersion, error) {
-	env, err := env.NewCoreForCI(ctx, log)
-	if err != nil {
+func getOpenShiftVersions() (*OpenShiftVersions, error) {
+	const envKey = "OPENSHIFT_VERSIONS"
```

Comment:
nit: would it be better to move this to a package-level const so it appears at the top of the file? Same for `INSTALLER_IMAGE_DIGESTS` below?

Reply:
+0, I can do that.

Reply:
It turns out we have a whole file (`/cmd/aro/const.go`) for environment variable names, but it needed a touch of rework.  See additional commits.